content/index.ko.md

```diff
@@ -112,30 +112,30 @@ footer: >   # Translate all the words below, including "Date:" and "Editor:". Do
 
 만약 접근성 기능이 한 요소에 잘 실행되지 않는다면 접근 가능한 사용자 경험을 야기하지 않게 되고, 다른 요소들이 접근성을 실행할 이유가 없어집니다. 예를 들어, 개발자는 저작 도구가 지원하지 않고, 대부분의 브라우저와 보조 기술이 계속해서 적용하지 않는 접근성 기능은 사용하지 않으려 할 것입니다. 
 
-{% assign example_url = "/fundamentals/components/examples/#weak" | relative_url %}
-![한 요소가 약할 경우에 대한 삽화. 자세한 설명은 다음 링크 참고 : {{ example_url }}]({{ "/content-images/wai-components/bridge.png" | relative_url }}){:longdesc="{{example_url}}"}
+{% assign longdesc_url = "/fundamentals/components/examples/#weak" | relative_url %}
+{% include image.html src="bridge.png" alt="한 요소가 약할 경우에 대한 삽화" longdesc=longdesc_url %}
 
 만약 한 요소가 접근성 지원이 제대로 이루어지지 않는다면, 다른 요소들이 더 많은 노력을 해야 하고 접근성 측면에서 전반적으로 좋지 않은 "차선책"을 통해 보완하고자 할 것입니다. 
```

Comment:
"For example" in the original version seems to be missing.

Reply:
Commit 2fed7bd